lib/jekyll-admin/server/page.rb

```diff
@@ -34,6 +39,18 @@ class Server < Sinatra::Base
 
       private
 
+      def html_content?
```

Comment:
We would need to write the file in order to use `Jekyll::Page` (we could do so via tmp files) but I was avoiding writing any files if it isn't going to be an html file. thoughts?